src/components/common/Directors.vue

```diff
@@ -524,7 +545,28 @@ import { CommonMixin, DateMixin, DirectorMixin, ResourceLookupMixin } from '@/mi
 import { Actions, FilingStatus } from '@/enums'
 
 // Interfaces
-import { FormIF, BaseAddressIF, AlertMessageIF } from '@/interfaces'
+import { FormIF, AddressIF, ComponentIF, AlertMessageIF } from '@/interfaces'
+
+// Interfaces used locally only (since they don't match global interfaces)
+interface OfficerIF {
```

Comment:
This interface and the one below are not used outside of this file so I just created them locally. Having them really helped with dev and debugging!